src/qtiCommonRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -98,8 +98,6 @@ var render = function render(interaction) {
                     renderTo: $('.media-container', $container)
                 })
                     .on('render', function() {
-                        resize();
```

Comment:
But when you change browsers window size, it still calls resize and brake all that you've done, correct?

Reply:
You are right, I push a fix